manifests/packages.pp

```diff
@@ -6,6 +6,12 @@
     ensure => $openssl::package_ensure,
   }
 
+  if $openssl::dev_package_ensure && $::osfamily == 'Debian' {
```

Comment:
I don't know if I like using parameters from another class like that, but that's how this class is written so...